### ISSUE TITLE
NTR: L-xylulose high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8866]
New term request L-xylulose high in urine (elthat_140410043612): L-xylulosuria (a form of pentosuria (pentose high in urine))

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000915
- label: Elevated urine L-xylulose level
- parents: [Abnormal urine carbohydrate level (HP:0031979)](https://hpo.jax.org/app/browse/term/HP:0031979)
- pmid: PMID:22042873
- def: The amount of L-xylulose in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8866
- synonyms:
